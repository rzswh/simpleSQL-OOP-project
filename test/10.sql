CREATE DATABASE qAz;
CREATE DATABASE eDc;
CREATE DATABASE iJn;
CREATE DATABASE aCs;
SHOW DATABASES;
USE qAz;
CREATE TABLE qwe(wer INT, ert INT, rty INT, tyu INT NOT NULL, PRIMARY KEY(tyu));
CREATE TABLE sqwe(wer INT, ert INT, rty INT, tyu INT NOT NULL, PRIMARY KEY(tyu));
INSERT INTO qwe(wer, ert, rty, tyu) VALUES (1, 2, 3, 4);
INSERT INTO qwe(wer, ert, rty, tyu) VALUES (1, 2, 3, 3);
INSERT INTO qwe(wer, ert, rty, tyu) VALUES (1, 2, 3, 2);
INSERT INTO qwe(wer, ert, rty, tyu) VALUES (1, 2, 3, 9);
INSERT INTO qwe(wer, ert, rty, tyu) VALUES (1, 2, 3, 10);
INSERT INTO qwe(wer, ert, rty, tyu) VALUES (1, 2, 3, 7);
INSERT INTO qwe(wer, ert, rty, tyu) VALUES (1, 2, 3, 8);
INSERT INTO qwe(wer, ert, rty, tyu) VALUES (1, 2, 3, 20);
INSERT INTO qwe(wer, ert, rty, tyu) VALUES (1, 2, 3, 30);
INSERT INTO qwe(wer, ert, rty, tyu) VALUES (1, 2, 3, 25);
UPDATE qwe SET wer=5 WHERE tyu>9;
UPDATE qwe SET ert=10 WHERE tyu<15;
SELECT * from qwe;
DELETE from qwe WHERE wer=5 and ert=10;
SELECT tyu from qwe;
SHOW columns from sqwe;
DROP DATABASE aCs;
DROP DATABASE qAz;
DROP DATABASE eDc;
DROP DATABASE iJn;
