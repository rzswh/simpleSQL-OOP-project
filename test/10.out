Database
aCs
eDc
iJn
qAz
wer	ert	rty	tyu
1	10	3	2
1	10	3	3
1	10	3	4
1	10	3	7
1	10	3	8
1	10	3	9
5	10	3	10
5	2	3	20
5	2	3	25
5	2	3	30
tyu
2
3
4
7
8
9
20
25
30
Field	Type	Null	Key	Default	Extra
wer	int(11)	YES		NULL	
ert	int(11)	YES		NULL	
rty	int(11)	YES		NULL	
tyu	int(11)	NO	PRI	NULL	
